.github/workflows/verify.yml

```diff
@@ -6,12 +6,16 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
-        with:
-          version: latest
-          only-new-issues: true
+    - uses: actions/checkout@v3
```

Comment:
I think this should be indented one more level, and the codespell job too. It might be the reason they're not triggering.